src/osd_window.rs

```diff
@@ -109,7 +109,7 @@ impl SwayosdWindow {
 		// Set the window margin
 		window.connect_map(clone!(@strong monitor => move |win| {
 			let bottom = monitor.workarea().height() - win.height_request();
-			let margin = (bottom as f32 * 0.75).round() as i32;
+			let margin = (bottom as f32 * get_top_margin()).round() as i32;
```

Comment:
Setting it to 0 actually doesn't put it on the top due to my bar being there. Could you add `gtk_layer_shell::set_exclusive_zone(&window, -1);` above line 43?